docs/react/use-custom-date-library.zh-CN.md

```diff
@@ -117,11 +117,63 @@ module.exports = {
 编写如下代码:
 
 ```tsx
-import dateFnsGenerateConfig from 'rc-picker/es/generate/dateFns';
 import generatePicker from 'antd/es/date-picker/generatePicker';
 import 'antd/es/date-picker/style/index';
+import dateFnsGenerateConfig from 'rc-picker/es/generate/dateFns';
 
 const DatePicker = generatePicker<Date>(dateFnsGenerateConfig);
 
 export default DatePicker;
 ```
+
+## 使用 luxon
+
+可以使用 [luxon](https://moment.github.io/luxon/) 代替 dayjs 并支持相同的功能，但它确实引入了一些我们将在下面解释的行为差异。
```

Comment:
```suggestion
可以使用 [luxon](https://moment.github.io/luxon/) 代替 dayjs 并支持同样的功能，但它与 dayjs 有一些差异，我们将在下面解释：
```

Reply:
Thank you 🙏 Adjusted in this commit: b9e00ebb8f800723c3da2e02e40b1adace1925c5